[0m23:15:24.285511 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024056AF2670>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024058B9FFD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024059743310>]}


============================== 23:15:24.295512 | e122df00-6950-4611-9ad7-2a63d3e53c6e ==============================
[0m23:15:24.295512 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:15:24.295512 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Levi\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m23:15:38.993471 [debug] [MainThread]: Starter project path: c:\levi\gyakorlatok\dbt_projects\env\lib\site-packages\dbt\include\starter_project
[0m23:15:39.011471 [info ] [MainThread]: 
Your new dbt project "dbt_tutorial" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m23:15:39.014470 [info ] [MainThread]: Setting up your profile.
[0m23:19:08.341409 [info ] [MainThread]: Profile dbt_tutorial written to C:\Users\Levi\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m23:19:08.345373 [debug] [MainThread]: Command `dbt init` succeeded at 23:19:08.343373 after 224.16 seconds
[0m23:19:08.346372 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024056AF2670>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000240596F0580>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002405A4A5DF0>]}
[0m23:19:08.347374 [debug] [MainThread]: Flushing usage events
[0m23:30:00.819559 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DA7D12610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DA888E100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DAA94ED90>]}


============================== 23:30:00.834406 | 93336ca0-3b4e-4754-91d6-6b06698c8a45 ==============================
[0m23:30:00.834406 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:30:00.835406 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\Levi\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m23:30:00.835406 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Levi\Gyakorlatok\dbt_projects\dbt_tutorial\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:30:00.837406 [debug] [MainThread]: Command `dbt run` failed at 23:30:00.836409 after 0.14 seconds
[0m23:30:00.837406 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DA7D12610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DA888E100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022DAA4BD790>]}
[0m23:30:00.838406 [debug] [MainThread]: Flushing usage events
[0m23:30:11.256193 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016645C626A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016647D17340>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001664889D940>]}


============================== 23:30:11.260195 | 4205cfe6-74a1-4180-914e-180d567819bf ==============================
[0m23:30:11.260195 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:30:11.260195 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Levi\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:30:11.423825 [info ] [MainThread]: dbt version: 1.8.7
[0m23:30:11.423825 [info ] [MainThread]: python version: 3.8.10
[0m23:30:11.424827 [info ] [MainThread]: python path: c:\levi\gyakorlatok\dbt_projects\env\scripts\python.exe
[0m23:30:11.424827 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m23:30:13.350773 [info ] [MainThread]: Using profiles dir at C:\Users\Levi\.dbt
[0m23:30:13.351782 [info ] [MainThread]: Using profiles.yml file at C:\Users\Levi\.dbt\profiles.yml
[0m23:30:13.352890 [info ] [MainThread]: Using dbt_project.yml file at C:\Levi\Gyakorlatok\dbt_projects\dbt_tutorial\dbt_project.yml
[0m23:30:13.353891 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m23:30:13.354892 [debug] [MainThread]: Command `dbt debug` failed at 23:30:13.354892 after 2.20 seconds
[0m23:30:13.354892 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016645C626A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016648891BE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000166488860A0>]}
[0m23:30:13.356047 [debug] [MainThread]: Flushing usage events
